devo/plugin.spec.yaml

```
@@ -0,0 +1,161 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: devo
+title: devo
+description: Devo is the cloud-native logging and security analytics solution that delivers real-time visibility for security and operations teams
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+cloud_ready: false
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/devo
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- devo
+- siem
+- logs
+- cloud
+hub_tags:
+  use_cases: [threat_detection_and_response, reporting_and_analytics, data_utility, alerting_and_notifications]
+  keywords: [logs]
+  features: []
+
+types:
+  log_entry:
+    bytesTransferred:
+      title: "Bytes Transferred"
+      type: integer
+      description: "Bytes Transferred"
+      required: false
+    clientIpAddress:
+      title: "Client IP Address"
+      type: string
+      description: "Client IP address"
+      required: false
+    cookie:
+      title: "Cookie"
+      type: string
+      description: "Cookie"
+      required: false
+    eventdate:
+      title: "Event Date"
+      type: integer
+      description: "Event date"
+      required: false
+    method:
+      title: "Method"
+      type: string
+      description: "Method"
+      required: false
+    protocol:
+      title: "Protocol"
+      type: string
+      description: "Protocol"
+      required: false
+    referralUri:
+      title: "Referral URI"
+      type: string
+      description: "Referral URI"
+      required: false
+    statusCode:
+      title: "Status Code"
+      type: integer
+      description: "Status code"
+      required: false
+    timeTaken:
+      title: "Time Taken"
+      type: integer
+      description: "Time taken"
+      required: false
+    timestamp:
+      title: "Time stamp"
+      type: string
+      description: "Time stamp"
```

```suggestion
      description: "Timestamp"
```
Is there any specific format this will be output in?